src/main/resources/fragments/icons/envelope.scala.html

```diff
@@ -0,0 +1,4 @@
+@(classNames: List[String] = Nil) 
+@svg("enveloppe", 44, 32, viewBox = true, classNames = classNames){
```

Comment:
The file name's spelling is different from the spelling here 😉

Reply:
nice catch 👍